compiler-core/src/parse.rs

```diff
@@ -2686,8 +2686,26 @@ where
         let mut unqualified_values = vec![];
         let mut unqualified_types = vec![];
 
-        if self.maybe_one(&Token::Dot).is_some() {
-            let _ = self.expect_one(&Token::LeftBrace)?;
+        if let Some((dot_start, dot_end)) = self.maybe_one(&Token::Dot) {
+            let _ = self.expect_one(&Token::LeftBrace).map_err(|e| {
```

Comment:
This looks like it'll show the error when there's _any_ unexpected token, not specifically when the Python syntax is used.

Reply:
Sorry, I mispoke there. What I mean is that `import one/two.three` is not a Python import